dbms/src/Storages/DeltaMerge/ColumnFile/ColumnFileSetWithVectorIndexInputStream.h

```diff
@@ -69,7 +69,7 @@ class ColumnFileSetWithVectorIndexInputStream : public VectorIndexBlockInputStre
         , data_provider(data_provider_)
         , ann_query_info(ann_query_info_)
         , valid_rows(std::move(valid_rows_))
-        , vec_index_cache(context_.global_context.getLocalIndexCache())
+        , vec_index_cache(context_.global_context.getLightLocalIndexCache())
```

Comment:
The interesting part is that maybe ColumnFileSet should use heavy cache (because all in memory and cannot be evicted by system) and DMFile should use light cache (because it does not directly occupy memories)?

Reply:
fixed